editions/tw5.com/tiddlers/wikitext/Formatting in WikiText.tid

```diff
@@ -1,19 +1,34 @@
 caption: Formatting
 created: 20131205155959399
-modified: 20220513120211686
+modified: 20220726132504220
 tags: WikiText
 title: Formatting in WikiText
 type: text/vnd.tiddlywiki
 
+<style>
+.no-code-border > code {
+  border: initial;
+  background-color: initial;
+}
+</style>
 Available character formatting includes:
 
-* <code>&#96;backticks&#96;</code> for `code` (<<.icon $:/core/images/mono-line>>)
-** Alternatively, <code>&#96;&#96;double backticks allows &#96;embedded&#96; backticks&#96;&#96;</code><br><br>
-* `''bold''` for ''bold text'' (<<.icon $:/core/images/bold>>)<br><br>
-* `//italic//` for //italic text// (<<.icon $:/core/images/italic>>)<br><br>
-* `__underscore__` for __underscored text__ (<<.icon $:/core/images/underline>>)<br><br>
-* `^^superscript^^` for ^^superscripted^^ text (<<.icon $:/core/images/superscript>>)<br><br>
-* `,,subscript,,` for ,,subscripted,, text (<<.icon $:/core/images/subscript>>)<br><br>
-* `~~strikethrough~~` for ~~strikethrough~~ text (<<.icon $:/core/images/strikethrough>>)
+<div class="no-code-border">
+
+|Wikitext | Editor Button |HTML |Rendered Output |h
```

Comment:
Second col has centered headline text.

Maybe change third col heading "HTML" into "HTML equivalent" or "rendered HTML" or some such because the people reading this whole tid are typically beginners and may otherwise believe that they need to know this (i.e learn html)

Overall for the table, I imagine an English teacher complaining about using "2" instead of "Two" or "Double".

Reply:
>Second col has centered headline text.

That's intentional. The buttons are centred too. It looks better.

>Maybe change third col heading "HTML" into "HTML equivalent" or "rendered HTML" or some such because the people reading this whole tid are typically beginners and may otherwise believe that they need to know this (i.e learn html)

I think I'll remove that column and add the HTML output to the end of the tiddler. eg: For those who are interested. 

>"2" instead of "Two" or "Double".  

That's intentional 2 ... IMO the number is much more visible as the word two. It's especially needed for `''bold''`